angularjs-portal-home/src/main/webapp/my-app/layout/partials/home-toggle.html

```diff
@@ -5,6 +5,7 @@
       tooltip-trigger="mouseenter"
       tooltip-placement="top"
       tooltip-popup-delay="500"
+      onClick="_gaq.push(['_trackEvent', 'Widgets', 'List Mode', 'List Mode']);"
```

Comment:
Hmm.  I think the API is `_trackEvent, {category}, {action}, {label}, {value}` .  So I'd read this as a "Widgets" category event with action "List Mode" and label "List Mode"?

Would it be better to go for something more like  

```
onClick="_gaq.push(['_trackEvent', 'Widgets', 'mode-select', 'List Mode']);"
```

so that the widget mode selection would have the same `action` so that these can be compared by label?  I don't know that it makes much difference, just trying to think through how to use the API to organize the taxonomy of events.